src/components/Rate/Rate.style.jsx

```diff
@@ -16,7 +16,7 @@ export const StyledRateButton = styled(({
     ${mixinSecondColor}
     ${mixinSize}
 
-    min-width: ${({ gutter, size, count }) => (size * count) + (gutter * (count - 1))}px;
+    ${({ nowrap }) => nowrap && 'white-space: nowrap'};
```

Comment:
之前有遇過一個 bug 是 `&&` 的寫法可能產出下列 CSS：
```css
font-size:14px;
false;
.ant-rate-star-half {...}
```

```suggestion
    ${({ nowrap }) => nowrap ? 'white-space: nowrap' : ''};
```